spark/src/main/kotlin/com/adevinta/spark/components/bottomsheet/BottomSheet.md

```diff
@@ -0,0 +1,51 @@
+# Package com.adevinta.spark.components.bottomsheet
+
+[BottomSheet](https://spark.adevinta.com/1186e1705/p/67d41e-bottom-sheet/b/02056b)
+A bottom sheet is a UI component commonly used in mobile applications to present additional content
+or options from the bottom of the screen.
+It is a modal component that slides up from the bottom of the screen and covers the entire screen.
+
+#### BottomSheet
+
+```kotlin
+fun BottomSheet(
+    onDismissRequest: () -> Unit,
+    modifier: Modifier = Modifier,
+    showHandle: Boolean = true,
+    contentTopPadding: Dp, ,
+    sheetState: SheetState = rememberModalBottomSheetState(),
+    content: @Composable BoxScope.() -> Unit,
+)
+```
+
+# BottomSheet content with / No handle Example
+
+| Light                                              | Dark                                               |
+|----------------------------------------------------|----------------------------------------------------|
+| ![](../../images/bottomsheet_dark_with_handle.png) | ![](../../images/bottomsheet_dark_with_handle.png) |
+| ![](../../images/bottomsheet_light_no_handle.png)  | ![](../../images/bottomsheet_dark_no_handle.png)   |
+
+# BottomSheet content behind handle Example
+
+![](../../images/bottomsheet_content_behind_handle.png)
```

Comment:
Let's showcase how to do it by code 😃 
As a dev I'm interested on how to achieve this example

Reply:
:D Sure man